docs/compilers/CSharp/Compiler Breaking Changes - DotNet 10.md

```diff
@@ -357,3 +357,34 @@ class extension
 }
 ```
 
+## Extended partial members are now virtual
+
+***Introduced in Visual Studio 2022 version 17.15***
+
+We have fixed [an inconsistency](https://github.com/dotnet/roslyn/issues/77346)
+where interface members would not be implicitly `virtual` and `public` if they were also marked `partial`.
```

Comment:
I think this description is unclear because accessibility for methods isn't affected by the change